src/components/RoiCalculatorModal/RoiCard.tsx

```diff
@@ -90,13 +91,18 @@ const RoiCard: React.FC<RoiCardProps> = ({ earningTokenSymbol, calculatorState,
 
   const onEnterEditing = () => {
     setCalculatorMode(CalculatorMode.PRINCIPAL_BASED_ON_ROI)
+    setExpectedRoi('')
```

Comment:
this will reset the input before enter editing, is this an expected behaviour?

Reply:
Yes, Imho when user clicks button to edit It should see empty input field that can put input easier than first removing the previous roi.

Reply:
@memoyil it clears expectedRoi but it leaves all other data in place (i.e. Token value for USD). Let's leave it as is maybe. Users might want to change "1034.33$" to "1000$" 🤷 
![Screenshot 2021-11-18 at 9 56 54](https://user-images.githubusercontent.com/81824236/142374984-cfa1d2ed-0b0e-45b3-9afd-2629961763d2.png)

Reply:
Done